landingzones/lz-machinelearning/lz.bicep

```
@@ -383,6 +390,37 @@ module aksCluster '../../azresources/containers/aks/main.bicep' = {
   }
 }
 
+module appServicePlan '../../azresources/compute/web/app-service-plan-linux.bicep' = if (appServiceLinuxContainer.enabled) {
+  name: 'deploy-app-service-plan'
+  scope: rgCompute
+  params: {
+    name: appServicePlanName
+    skuName: appServiceLinuxContainer.skuName
+    skuTier: appServiceLinuxContainer.skuTier
+
+    tags: resourceTags
+  }
+}
+
+module appServiceLC '../../azresources/compute/web/appservice-linux-container.bicep' = if (appServiceLinuxContainer.enabled) {
+  name: 'deploy-app-service'
+  scope: rgCompute
+  params: {
+    name: appServiceLinuxContainerName
+    appServicePlanId: appServicePlan.outputs.planId
```

We should only reference the output if `appServiceLinuxContainer.enabled = true`.

```suggestion
    appServicePlanId: appServiceLinuxContainer.enabled ? appServicePlan.outputs.planId : ''
```